sysutils/kubectx/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ahmetb kubectx v0.6.3
+github.tarball_from archive
+
+name                kubectx
+version             0.6.3
+categories          sysutils
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+
+maintainers         @gearnode
```

Comment:
You will want to instead use something including your email address, like

`maintainers  {foo.com:bar @gearnode}`

And unless there's a good reason not to accept openmaintainer:

`maintainers  {foo.com:bar @gearnode} openmaintainer`